department-of-veterans-affairs/caseflow
# Fix disposition error

### Description
When we set remand reasons, we were previously erasing all other issues from the appeal.

### Acceptance Criteria
- Remand reasons are set without affecting other fields on the appeal and its issues

### Testing Plan
1. For both RAMP and legacy, navigate to attorney checkout, choose some remanded issues and some not remanded issues
1. choose remand reasons
1. navigate back and verify that the original issues/dispositions are still present
